src/ScheduledTasks/Client/ScheduleClientImpl.cs

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Client.Entities;
+using Microsoft.DurableTask.Entities;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DurableTask.ScheduledTasks;
+
+/// <summary>
+/// Represents a handle to a scheduled task, providing operations for managing the schedule.
+/// </summary>
+// TODO: Isolate system entity from user entities
+class ScheduleClientImpl : ScheduleClient
```

Comment:
nit: .NET convention is not to use "Impl" suffix, but to instead use "Default" prefix. So in this case, the class name would be `DefaultScheduleClient`.